### PR TITLE
Updated 3.16 and 3.17 release notes to indicate the promotion of 3.16…

### PR DESCRIPTION
**Pull request by ijlee2**
I made changes based on 3.8, 3.12-3.15 release notes.

### CODE REVIEW

File: source/2020-02-12-ember-3-16-released.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ tags: Releases, 2020, 3, 3.16, Version 3.x
 responsive: true
 ---
 
-Today the Ember project is releasing version 3.16 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.17 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+Today the Ember project is releasing version 3.16 of Ember.js, Ember Data, and Ember CLI. This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.
+
+This release kicks off the 3.17 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```
Comment (ijlee2) [inline source/2020-02-12-ember-3-16-released.md:10]: Same as before, but on a new paragraph.

File: source/2020-02-12-ember-3-16-released.md [hunk 2]
```diff
@@ -22,6 +24,10 @@ Ember.js is the core framework for building ambitious web applications.
 
 ### Changes in Ember.js 3.16
 
+The 3.16.0 release is an Ember.js Long-Term Support candidate. In six weeks, the 3.16.x series will become the latest LTS release and six weeks after that the 3.8 LTS branch will no longer receive bugfix patches. LTS versions allow teams to upgrade less frequently while still getting support from the Ember project and the wider ecosystem.
```
Comment (ijlee2) [inline source/2020-02-12-ember-3-16-released.md]: I wasn't sure if writing "six weeks after that the 3.8 LTS branch will no longer receive bugfix patches" is correct.

The 3.12 release note showed 3.4 in this part of the sentence, so I assumed the difference was always 0.8. However, the 3.8 release note also showed 3.4.

File: source/2020-03-16-ember-3-17-released.md [hunk 3]
```diff
@@ -91,7 +95,19 @@ There are no new deprecations in Ember CLI 3.17.
...
 
 For more details on the changes in Ember CLI 3.17 and detailed upgrade
-instructions, please review the [Ember CLI  3.17.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.17.0).
+instructions, please review the [Ember CLI 3.17.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.17.0).
+
+## Ember 3.16 is an LTS Release
+
+If you are upgrading from the previous LTS version (3.12) to 3.16 then you are getting access to these features:
+
+- Tracked properties (3.13)
```
Comment (ijlee2) [inline source/2020-03-16-ember-3-17-released.md:104]: I looked at 3.12-3.15 release notes to gather features that impact app developers (and omitted features for addon developers).

File: source/2020-02-12-ember-3-16-released.md (hunk 2 above)
Comment (kennethlarsen) [reply source/2020-02-12-ember-3-16-released.md]:
```suggestion
The 3.16.0 release is an Ember.js Long-Term Support candidate. In six weeks, the 3.16.x series will become the latest LTS release and six weeks after that the 3.12 LTS branch will no longer receive bugfix patches. LTS versions allow teams to upgrade less frequently while still getting support from the Ember project and the wider ecosystem.
```

That must have been a mistake. It should be `3.12` in this case.

File: source/2020-03-16-ember-3-17-released.md (hunk 3 above)
Comment (kennethlarsen) [reply source/2020-03-16-ember-3-17-released.md:104]: Really awesome, thanks!

**Comment by kennethlarsen** [comment]
@ijlee2 Thanks a lot - this is really awesome. I added one minor correction.

**Comment by kennethlarsen** [comment]
@ijlee2 Oh, and please add your self as an author to both posts :)

File: source/2020-02-12-ember-3-16-released.md (hunk 2 above)
Comment (ijlee2) [reply source/2020-02-12-ember-3-16-released.md]: Ah, ok.

**Review by kennethlarsen — APPROVED** [review]
Thank you so much ✨

**Pull request merged**